plugins/modules/cobbler_system.py

```diff
@@ -161,6 +161,9 @@
 from ansible_collections.community.general.plugins.module_utils.datetime import (
     now,
 )
+from ansible_collections.community.general.plugins.module_utils import deps
+with deps.declare("packaging"):
+    from packaging.version import Version
```

Comment:
Adding a mandatory dependency to the module is a breaking change and won't get merged. Please use `LooseVersion` from `ansible_collections.community.general.plugins.module_utils.version`.